src/resources/common/enums.ts

```diff
@@ -142,12 +142,32 @@ export enum OnlineBrand {
     BOOST = "boost",
     RABBIT_LINE_PAY = "rabbit_line_pay",
 
+    // Alipay+ Online
+    TINABA = "tinaba",
+    NAVER_PAY = "naver_pay",
+    TOSS_PAY = "toss_pay",
+    PAYMAYA = "paymaya",
+    GRAB_PAY = "grab_pay",
+    KREDIVO = "kredivo",
+
     /**
      * @deprecated Use `ALIPAY_PLUS_ONLINE` instead
      */
     ALIPAY_CONNECT_ONLINE = "alipay_connect_online",
 }
 
+// Alipay+ Offline
+export enum RetailsBrands {
```

Comment:
should we go with `RetailBrands` to be consistent with this naming pattern that we use plural only for the last word?

Reply:
Indeed, thanks for catching that! 👍